### ISSUE TITLE
Don't allow multiple users with the same name.

### ISSUE THREAD

**Issue by Tejpbit** [Kerp1/Hastings#61]
This i probably easiest implemented when we have database.
So first fix #47, Then fix #37, Then this issue should be an easy fix.

**Comment by mattiasnilsen** [comment]
This is fixed in #84

**Comment by Kalior** [comment]
I would like to debate this. I think it is possible to sign in with the same name as someone else but not change to a name that someone else has. Am I wrong?

**Comment by mattiasnilsen** [comment]
No that's correct but I think that's another problem that has more to do with the issue regarding user authentication. However we should probably prevent the same user from signing in twice since right now the same user will be given two different sessionIDs and thus be considered as two users in some cases and as one user in other.

**Comment by Kalior** [comment]
I agree that is also a problem. However, in the current system, multiple users with the same name are allowed, which is what this issue is about. 

I would like to see an implementation that checks the value of the prompt if that user name is taken and in that case it prompts again.

**Comment by mattiasnilsen** [comment]
Fair enough, issue is reopened.
